src/components/feature/settings/UpdateUserNameForm/index.tsx

```
@@ -42,7 +42,40 @@ export function UpdateUserNameForm({ currentUser }: UpdateUserNameFormProps) {
   return (
     <form onSubmit={handleSubmit} className="space-y-4">
       <div className="flex items-center space-x-4">
-        <UserIcon userImage={currentUser.image} size="md" />
+        <UploadImageWrapper
```

ESLint

[error] 'UploadImageWrapper' is not defined.

(react/jsx-no-undef)